client/app/(authenticated)/dashboard/access-requests/page.tsx

```diff
@@ -0,0 +1,19 @@
+// import Link from "next/link";
+// import { Button } from "@mui/material";
+import { Suspense } from "react";
+import AccessRequests from "@/app/components/routes/access-requests/AccessRequests";
+import Loading from "@/app/components/loading/SkeletonGrid";
+
+export default function Page() {
+  return (
+    <>
+      <h1>Operator Admin Access Requests List</h1>
+      {/* <Link href="/dashboard/operations/create">
+        <Button variant="contained">Add Operation</Button>
+      </Link> */}
+      <Suspense fallback={<Loading />}>
+        <AccessRequests />
+      </Suspense>
+    </>
```

Comment:
I think the wireframe might have changed since this went up for review; now there's no header and there's a note

Reply:
updated